File: BaekJoon/10xx/1002.py
import math

num = int(input())

inputList = []

for i in range(num):
    inputList.append(list(map(int,input().split())))

for i in range(num):
    x1, y1, r1, x2, y2, r2 = inputList[i]

    distance = math.sqrt(math.pow(x1-x2,2)+math.pow(y1-y2,2))

    if distance == 0 and r1 == r2:
        print(-1)
    elif math.fabs(r1-r2) < distance and distance < r1+r2 :
        print(2)
    elif distance == r1+r2 or distance == math.fabs(r1-r2):
        print(1)
    else:
        print(0)

File: BaekJoonSwift/BaekJoonSwift/Basic Math2/1002.swift
import Foundation

let n = Int(readLine()!)!

for _ in 0..<n {
    let input = readLine()!.split(separator: " ").map { Double($0)! }
    let d = sqrt((pow((input[0] - input[3]) , 2) + pow((input[1] - input[4]) , 2)))
    let r1 = input[2]
    let r2 = input[5]
    
    if d == 0 && r1 == r2{
        print(-1)
    } else if fabs(r1-r2) < d && d < r1+r2 {
        print(2)
    } else if d == r1+r2 || d == fabs(r1-r2){
        print(1)
    } else {
        print(0)
    }
}

